### DISCUSSION TITLE
batch file that i made will anyone find this useful?

### DISCUSSION THREAD

**Discussion by DragonVsKira** [anidl/multi-downloader-nx#179] [Show and tell]
![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/35433196/144585539-acd08d23-80b8-4e7e-8812-8eba2d03f73d.gif)
>
**Let me know ?**

**up to date picture** 

<img width="851" alt="Screenshot 2021-12-03 020627" src="https://user-images.githubusercontent.com/35433196/144587066-8af4e883-4a8c-4e13-b17e-cbbcd36a2f89.png">

it only work with "funimation" for now if yes ill see @izu-co can add it to the master

@izu-co be4 u say anything u sad "ideas" can be post here this is my "ideas" lol

**Comment by izu-co** [comment]
@DragonVsKira 
Can you let me see it in action?
If you can keep this batch file up to date, I can add it to the repo. Also, I thought about making a GUI, but that would require more work. Maybe you are up for it :).

**Comment by DragonVsKira** [comment]
i had a gui if u forget lol but when u did this "multi-downloader-nx" i have to re do alot of code so i made this ...i have not did "crunchy" because there alot to do there 

i can change any thing just let me know  but here u go not my best work on making a video ok
https://drive.google.com/file/d/1JYIMDagUKUIygJ8r5B6Yb8equ_lhv0Qx/view?usp=sharing

**Comment by DAREK0N** [comment]
That's a nice idea tbh, but it's just another way of doing the same thing.

**Comment by DragonVsKira** [comment]
will it more to help new comer that do not know what a CMD is there alot of "issues" the r people doing the CMD wrong or not logging  in there account not for pro but pro can use it to lol it just a another way to use it @DAREK0N  that all trying to make it user-friendly that all

**Comment by DragonVsKira** [comment]
did u get time to look at the video @izu-co just asking

**Reply by izu-co** [reply]
Yes I had. Can you speed up the print in the beginning? I think it would be really annoying to wait. 
The rest seems fine (some English mistakes, tho :)):

**Reply by DragonVsKira** [reply]
**i can make it show "multi-downloader-nx" the first it at menu then when u go back to the menu it look like this**

<img width="855" alt="Screenshot 2021-12-05 040657" src="https://user-images.githubusercontent.com/35433196/144746004-6bbe76d8-86f7-4d90-adf4-a1f695f483d5.png">

**that the best i can do  For speed but if u reopen it will show "multi-downloader-nx" again or i can remove it but it look cool lol**

**Reply by shoyrock** [reply]
can you please share this it would make my life much easier

**Reply by DragonVsKira** [reply]
> can you please share this it would make my life much easier

i can but u will need to modify some code to my it work more for u as it a wip if that ok hit me up on discord DRAGONVSKIRA#9308
